ec-cpp/f2e16.hpp

```diff
@@ -0,0 +1,96 @@
+//
+// Created by iceseer on 5/4/23.
+//
+
+#ifndef NOVELPOLY_REED_SOLOMON_CRUST_F2E16_HPP
+#define NOVELPOLY_REED_SOLOMON_CRUST_F2E16_HPP
+
+#include <array>
+#include <assert.h>
+#include <cstdint>
+#include <stdlib.h>
+#include <tuple>
+#include <vector>
+
+#include "additive_fft.hpp"
+#include "errors.hpp"
+#include "math.hpp"
+#include "poly_encoder.hpp"
+#include "types.hpp"
+#include "walsh.hpp"
+
+namespace ec_cpp {
+
+struct f2e16_Descriptor {
+  using Elt = uint16_t;
```

Comment:
Hm, what's elt? Probably better to add a comment, since at least I am unfamiliar with this name.